applications/GeoMechanicsApplication/tests/test_k0_procedure_process/README.md

```diff
@@ -1,12 +1,13 @@
-- [K<sub>0</sub> procedure normal consolidation with skew layers](https://github.com/KratosMultiphysics/Kratos/tree/master/applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_skew_layers/README.md)
-- [K<sub>0</sub> procedure normal consolidation with skew layers and dam](https://github.com/KratosMultiphysics/Kratos/tree/master/applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_skew_layers_dam/README.md)
-- [K<sub>0</sub> procedure with umat](https://github.com/KratosMultiphysics/Kratos/tree/master/applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_umat/README.md)
-- [K<sub>0</sub> procedure for simple dike](https://github.com/KratosMultiphysics/Kratos/tree/master/applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_simple_dike/README.md)
-
+-   [K<sub>0</sub> procedure normal consolidation](https://github.com/KratosMultiphysics/Kratos/tree/master/applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc/README.md)
+-   [K<sub>0</sub> procedure normal consolidation with layers](https://github.com/KratosMultiphysics/Kratos/tree/master/applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_layers/README.md)
+-   [K<sub>0</sub> procedure normal consolidation with OCR](https://github.com/KratosMultiphysics/Kratos/tree/master/applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_ocr/README.md)
+-   [K<sub>0</sub> procedure normal consolidation with skew layers](https://github.com/KratosMultiphysics/Kratos/tree/master/applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_skew_layers/README.md)
+-   [K<sub>0</sub> procedure normal consolidation with skew layers and dam](https://github.com/KratosMultiphysics/Kratos/tree/master/applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_skew_layers_dam/README.md)
+-   [K<sub>0</sub> procedure with umat](https://github.com/KratosMultiphysics/Kratos/tree/master/applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_umat/README.md)
+-   [K<sub>0</sub> procedure for simple dike](https://github.com/KratosMultiphysics/Kratos/tree/master/applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_simple_dike/README.md)
+-   [K<sub>0</sub> procedure for horizontal layers](https://github.com/KratosMultiphysics/Kratos/tree/master/applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_with_horizontal_layers/README.md)
+-   [K<sub>0</sub> procedure for tilted layers](https://github.com/KratosMultiphysics/Kratos/tree/master/applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_with_tilted_layers/README.md)
```

Comment:
Shouldn't we make changes to all the k0 tests which were there before to explicitly set it the 'use-standard-procedure' to false? Or do all tests pass now?

Reply:
Since all tests pass, I decided to not add the "use_standard_procedure" flag to the existing test files. But I will double-check with @WPK4FEM to make sure that I did not make any mistake here.

Reply:
Anne has checked today.